P-7155: Added SLA webhook doc (#48)
team-plain/docs: pages/threads.mdx

# Threads

Threads are the core of Plain's data model and equivalent to tickets or conversations in other support platforms. When you use Plain to help a customer you assign yourself to a thread and then mark the thread as done once you're done helping.

## Creating a thread

They are created in 3 ways:

- [Email](./email-setup): When an email is sent to your support address, a new thread is created, or if a thread already exists for this email chain, the email is added to the existing thread.
- [Contact Forms](./contact-forms-setup): When a contact form is submitted you can use the Plain API to create a new thread.
- [Create Thread API](./graphql/threads): You can also create a thread via the API when you want to react to something happening in your systems and provide proactive support before the customer even reaches out to you.

## Adding context to a thread

> Plain is built API-first which means that [app.plain.com](https://app.plain.com) is built using the same API we make available to you. This means you can add context to a thread when events, webhooks, or any event you can programatically
> listen to happens in your systems.

Reviewer: ```suggestion
With Plain's API you can also provide important additional context to any thread so that you can offer fast support and always know who you are talking to, what happened, and why.

You can do so in 3 ways:
```

# Threads

Threads are the core of Plain's data model and equivalent to tickets or conversations in other support platforms. When you use Plain to help a customer you assign yourself to a thread and then mark the thread as done once you're done helping.

## Creating a thread

> They are created in 3 ways

Reviewer: ```suggestion
They are created in 3 ways:
```

- [Email](./email-setup): When an email is sent to your support address, a new thread is created, or if a thread already exists for this email chain, the email is added to the existing thread.

> - [Contact Forms](./contact-forms-setup): A commmon use case for starting a thread is when a contact form is submitted,the API can be used to create a new thread.

Reviewer: ```suggestion
- [Contact Forms](./contact-forms-setup): When a contact form is submitted you can use the Plain API to create a new thread.
```

- [Create Thread API](./graphql/threads): You can also create a thread via the API when you want to react to something happening in your systems and provide proactive support before the customer even reaches out to you.

## Adding context to a thread

With Plain's API you can also provide important additional context to any thread so that you can offer fast support and always know who you are talking to, what happened, and why.

You can do so in 3 ways:

> - [Events](./events): can be used to log key events for a customer, across all or specific threads for a customer.

Reviewer: ```suggestion
- [Events](./events): can be used to log key events across all or specific threads for a customer.
```

> - [Customer Cards](./customer-cards): expose data that is fecthed from your own systems while looking at a thread in Plain.

Reviewer: ```suggestion
- [Customer Cards](./customer-cards): see important context and data directly from your systems when viewing any thread in Plain
```

> - [Labels](./labels): can be added to threads via the API, or the UI, to categorize threads and inform insights.

Reviewer: ```suggestion
- [Labels](./labels): can be added to threads via the API, or the UI, to categorize threads and help you quickly know what your support requests are about.
```

Threads are the core of Plain's data model and equivalent to tickets or conversations in other support platforms. When you use Plain to help a customer you assign yourself to a thread and then mark the thread as done once you're done helping.

## Creating a thread

> They are created in 3 ways

Reviewer 2: They are created in 3 ways:
